vendor/golang.org/x/text/internal/BUILD

```diff
@@ -9,7 +9,6 @@ go_library(
     ],
     importpath = "golang.org/x/text/internal",
     visibility = ["//visibility:public"],
-    deps = ["//vendor/golang.org/x/text/language:go_default_library"],
```

Comment:
Is this something we need to codify somewhere? This build file will get blown away the next time we dep ensure, as dep assumes ownership of `vendor/` and deletes it completely when `dep ensure` is run.

Reply:
yes, definitely. I mostly included this commit here as documentation for "how did I even make this work", since the scripts don't handle that right now.